packages/subgraph/.env-example

```diff
@@ -1,5 +1,4 @@
 NETWORK_NAME=< arbitrum-rinkeby || rinkeby || localhost ...etc >
 GRAPH_KEY=< subgraph key >
-THEGRAPH_USERNAME=< aragon >
 SUBGRAPH_NAME=< zaragoza >
```

Comment:
why we removed THEGRAPH_USERNAME ? if not needed, disregard/resolve my comment. Just making sure you mistakenly didn't remove it.

Reply:
not needed